src/nodedef.h

```diff
@@ -422,4 +427,3 @@ class NodeResolver {
 
 
 #endif
-
```

Comment:
Don't remove newlines, they serve a purpose (I don't know exactly which, but @ShadowNinja wants them in).